contrib/zenith/inmem_smgr.c

```diff
@@ -195,6 +195,10 @@ inmem_nblocks(SMgrRelation reln, ForkNumber forknum)
 {
 	int			nblocks = 0;
 
+	/*
```

Comment:
#0  inmem_nblocks (reln=0x5620695fc048, forknum=VISIBILITYMAP_FORKNUM)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/contrib/zenith/inmem_smgr.c:197
#1  0x000056206894beba in smgrnblocks (reln=0x5620695fc048, forknum=forknum@entry=VISIBILITYMAP_FORKNUM)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/storage/smgr/smgr.c:564
#2  0x00005620686b5aba in vm_extend (rel=rel@entry=0x5620695dcdd8, vm_nblocks=vm_nblocks@entry=1)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/access/heap/visibilitymap.c:650
#3  0x00005620686b5c44 in vm_readbuf (rel=rel@entry=0x5620695dcdd8, blkno=blkno@entry=0, extend=extend@entry=true)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/access/heap/visibilitymap.c:576
#4  0x00005620686b5ea3 in visibilitymap_pin (rel=rel@entry=0x5620695dcdd8, heapBlk=<optimized out>, buf=buf@entry=0x7ffdf3e751f8)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/access/heap/visibilitymap.c:202
#5  0x000056206869dd6a in heap_xlog_insert (record=0x5620695e5928)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/access/heap/heapam.c:8904
#6  0x0000562068956e6d in ApplyRecord (input_message=0x7ffdf3e77270)
    at /home/knizhnik/zenith/tmp_install/build/../../vendor/postgres/src/backend/tcop/zenith_wal_redo.c:612